bedrock/firefox/views.py

```diff
@@ -412,16 +412,19 @@ class WhatsnewView(L10nTemplateView):
         "firefox/whatsnew/whatsnew-fx117-de-reader-view.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx117-fr-reader-view.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx117-uk-reader-view.html": ["firefox/whatsnew/whatsnew"],
+        "firefox/whatsnew/whatsnew-fx117-vpn.html": ["firefox/whatsnew/whatsnew"],
```

Comment:
Do we need to make sure this template is active for all the locales it contains hard-coded strings for?

Reply:
I've wrapped them all in an `if ftl_file_is_active("firefox/whatsnew/whatsnew")` but not sure if that's the best way to do it.... I started putting that in the view but then realized we still needed a way to fall back to English or to another template and the view is already pretty hairy, so it seemed cleaner to check for activation in the template itself. Maybe I'm wrong?

Reply:
Hmm, then what would locales see if the file isn’t active? Would they see in the template and what would the locale be in the URL?

I was more thinking you could set `active_locales` in the view’s context so we can be sure the locales we want are active in prod. A bit like we did here https://github.com/mozilla/bedrock/commit/424aa8783a9000e28e64d89a4f445dd1cf3cc3d3

I’m not sure what’s best to be honest, in terms of this staying simple 😕

Reply:
The locales are all active in prod, they just don't have the `firefox/whatsnew/whatsnew` file completed. And it seems they *already* show English or mixed English even on the older fallback WNP. E.g. Lithuanian: https://www.mozilla.org/lt/firefox/116.0/whatsnew/ and Latvian: https://www.mozilla.org/lv/firefox/116.0/whatsnew/. 

So in that case it seems like even mixed languages is better than nothing and we shouldn't do anything else special? In the case of Latvian this will be the first localized WNP they've seen in years.

Reply:
If we’re confident that all the needed locales are already enabled for this I’ll in prod, then yeah nothing else should be needed 👍